test/error_messages.coffee

```diff
@@ -989,3 +991,39 @@ test "`&&=` and `||=` with a space in-between", ->
     a or = 1
          ^
   '''
+
+test "anonymous functions cannot be exported", ->
+  assertErrorFormat '''
+    export ->
+      console.log 'hello, world!'
+  ''', '''
+    [stdin]:1:8: error: unexpected ->
+    export ->
+           ^^
+  '''
+
+test "anonymous classes cannot be exported", ->
+  assertErrorFormat '''
+    export class
+      @constructor: ->
+        console.log 'hello, world!'
+  ''', 'SyntaxError: anonymous classes cannot be exported'
```

Comment:
I'm not sure if this is "your fault" or if it belongs to this PR, but it's sad that this error is so ugly (no `file:line:col` and no code snippet with `^^^` marker) compared to the other ones here ...